src/en/zinchanmanga/src/eu/kanade/tachiyomi/extension/en/zinchanmanga/ZinChanManga.kt

```diff
@@ -28,7 +29,8 @@ class ZinChanManga : HttpSource() {
     private val apiClient by lazy {
         network.client.newBuilder()
             .sslSocketFactory(ZinChanCert.factory, ZinChanCert.manager)
-            .addInterceptor(RateLimitInterceptor(2)).build()
+            .addInterceptor(RateLimitInterceptor(permits = 5, period = 50, TimeUnit.SECONDS))
```

Comment:
Seconds is the default unit.

```suggestion
            .addInterceptor(RateLimitInterceptor(5, 50))
```